packages/grafana-ui/src/components/Select/Select.tsx

```diff
@@ -251,6 +256,7 @@ export class AsyncSelect<T> extends PureComponent<AsyncProps<T>> {
               }}
               defaultValue={defaultValue}
               value={value}
+              hideText={hideTextValue}
```

Comment:
this is the one that I don't think is used, as there is no such property on this component.